resources/themes/cura-dark/theme.json

```diff
@@ -5,7 +5,7 @@
     },
 
     "colors": {
-        "main_background": [39, 44, 48, 255],
+        "main_background": [31, 31, 32, 255],
```

Comment:
I did this to see the buttons in the same way as the background of the design was.

However, since the buttons are never directly on the scene, I'll put this back (since it wasn't really doing what I want either).

Reply:
Resolved in: https://github.com/Ultimaker/Cura/pull/10071/commits/a42fee48340f72842e0a340fc090321eff71ed23